Tasks/KubernetesManifestV0/src/actions/bake.ts

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+import tl = require('vsts-task-lib/task');
+import path = require('path');
+import fs = require('fs');
+import * as toolLib from 'vsts-task-tool-lib/tool';
+import * as os from "os";
+import * as util from "util";
+import { WebRequest, sendRequest, getTempDirectory } from "../utilities";
+const uuidV4 = require('uuid/v4');
+const helmToolName = "helm"
+const helmLatestReleaseUrl = "https://api.github.com/repos/helm/helm/releases/latest";
+const stableHelmVersion = "v2.9.1"
+
+export async function bake() {
+    let helmPath = await getHelm();
+    let helmCommand = tl.tool(helmPath);
+    helmCommand.arg("template");
+    helmCommand.arg(tl.getPathInput("chart"))
+    let args = tl.getDelimitedInput("overrides", "\n");
+    helmCommand.arg(setArgs(args));
+    var result = helmCommand.execSync();
+    let pathToBakedManifest = getTemplatePath(result.stdout);
+    tl.setVariable(tl.getInput("manifestsBundle"), pathToBakedManifest);
+}
+
+function getTemplatePath(data) {
+    var paths = path.join(getTempDirectory(), "baked-template" + uuidV4() + ".yaml");
+    fs.writeFileSync(paths, data)
+    return paths;
+}
+
+function setArgs(args) {
+    var newArgs = [];
+    args.forEach(arg => {
+        let a = arg.split(":");
+        newArgs.push("--set")
```

Comment:
why are we supporting --set-file?

Reply:
It's not --set-file
It is more like `--set key=value`